FlowCryptAppTests/Functionallity/OrganisationalRulesTests.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  OrganisationalRulesTests.swift
+//  FlowCryptAppTests
+//
+//  Created by Anton Kharchevskyi on 17.09.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+import XCTest
+@testable import FlowCrypt
+
+class OrganisationalRulesTests: XCTestCase {
+    var sut: OrganisationalRules! {
```

Comment:
What does `sut` here mean?

Reply:
In another PR, we should refactor the naming. It's not clear what's the benefit of having an `OrganisationalRules` object that only wraps `ClientConfiguration` object. We should just have a single `ClientConfiguration` object and that's all. If we do need a wrapper for some justified reason, then we can call that `ClientConfigurationWrapper` and add a comment on why it's needed. If it's to match it to a http response, then we should call it `ClientConfigurationResponse` that contains `ClientConfiguration`.

Reply:
System under test. 
This is pretty common to use this naming in iOS unit tests

https://en.wikipedia.org/wiki/System_under_test

Reply:
got it. that's ok